tests/testthat/_snaps/fledgling.md

```diff
@@ -12,7 +12,7 @@
           },
           "section_state": "keep",
           "title": "fledge v2.0.0",
-          "version": "2.0.0",
+          "version": "v2.0.0",
```

Comment:
do we actually want to clean this?

Reply:
@krlmlr what did you expect here, should the version stored by fledgeling keep the v or not?

Reply:
No preference.

Reply:
The fledgeling object should be opaque, the implementation is more interesting.

Reply:
Rethinking this. There's reason to leave the `"v"` out because then it can be easier converted to a numeric version.

``` r
base::numeric_version("v1.2.3")
#> Error: invalid version specification 'v1.2.3'
base::numeric_version("1.2.3")
#> [1] '1.2.3'
```

<sup>Created on 2023-05-27 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>